### ISSUE TITLE
Give more edge info in the Edge Explorer.

### ISSUE THREAD

**Issue by maximusunc** [NCATS-Gamma/robokop#321]
Currently in the Edge Explorer modal, for edges we only show their source database. It has been requested that we try and provide more info that we have to the user.

**Comment by maximusunc** [comment]
One idea mentioned in our meeting was to try and provide a link to the database with specific information for that specific edge.

On the UI side, we're only given information similar to this: (note: I've shortened the publications and equivalent identifiers just to make this a little more reasonable)
`{
  "nodes": [
    {
      "name": "NPC1",
      "location": "18q11.2",
      "locus_group": "protein-coding gene",
      "gene_family": [
        "Solute carriers"
      ],
      "chromosome": "18",
      "id": "HGNC:7897",
      "equivalent_identifiers": [
        ...
      ],
      "taxon": "9606",
      "gene_family_id": [
        752
      ],
      "type": "gene",
      "isSet": false,
      "binding": "n1"
    },
    {
      "name": "sphingolipidosis",
      "nutritional_or_metabolic_disease": true,
      "id": "MONDO:0019255",
      "equivalent_identifiers": [
        "UMLS:C0037899",
        "MESH:D013106",
        "DOID:1927",
        "MONDO:0019255",
        "ORPHANET:79225"
      ],
      "type": "genetic_condition",
      "isSet": false,
      "binding": "n2"
    }
  ],
  "edges": [
    {
      "relation": [
        "RO:0002200"
      ],
      "target_id": "HGNC:7897",
      "edge_source": [
        "biolink.disease_get_gene"
      ],
      "publications": [],
      "id": "18010190",
      "predicate_id": "RO:0002200",
      "source_database": [
        "biolink"
      ],
      "source_id": "MONDO:0019255",
      "type": "has_phenotype",
      "ctime": [
        1549667730.8836133
      ],
      "relation_label": [
        "has phenotype"
      ],
      "binding": "e1"
    },
    {
      "relation": [
        "GENO:0000840"
      ],
      "target_id": "HGNC:7897",
      "edge_source": [
        "biolink.disease_get_gene"
      ],
      "publications": [
        ...
      ],
      "id": "18010279",
      "predicate_id": "BIOLINK:0000001",
      "source_database": [
        "biolink"
      ],
      "source_id": "MONDO:0019255",
      "type": "gene_associated_with_condition",
      "ctime": [
        1549667730.888653
      ],
      "relation_label": [
        "pathogenic_for_condition"
      ],
      "binding": "e1"
    },
    {
      "publications": [
        ...
      ],
      "type": "literature_co-occurrence",
      "source_database": "omnicorp",
      "source_id": "HGNC:7897",
      "target_id": "MONDO:0019255",
      "id": "BkW605oY3E"
    }
  ]
}`

**Comment by cbizon** [comment]
Yes, I think that a big part of the effort here would be to go back through our data sources and determine, for each, what would be a good way to construct an edge-based linkout, and then eithe provide a service to get those or add a property to the edges...

**Comment by patrickkwang** [comment]
The GUI currently displays all info available from the KG. If we want some linkout, we can add it to the edge properties when building/crawling.
